process/elasticproc/block/blockProcessor_test.go

```diff
@@ -60,30 +59,38 @@ func TestBlockProcessor_PrepareBlockForDBShouldWork(t *testing.T) {
 
 	bp, _ := NewBlockProcessor(&mock.HasherMock{}, &mock.MarshalizerMock{})
 
-	dbBlock, err := bp.PrepareBlockForDB(
-		[]byte("hash"),
-		&dataBlock.Header{},
-		[]uint64{0, 1, 2},
-		&dataBlock.Body{
-			MiniBlocks: dataBlock.MiniBlockSlice{
-				{
-					ReceiverShardID: 1,
-				},
-				{
-					ReceiverShardID: 2,
+	outportBlockWithHeader := &outport.OutportBlockWithHeader{
+		Header: &dataBlock.Header{},
+		OutportBlock: &outport.OutportBlock{
+			BlockData: &outport.BlockData{
+				HeaderHash: []byte("hash"),
+				Body: &dataBlock.Body{
+					MiniBlocks: dataBlock.MiniBlockSlice{
+						{
+							ReceiverShardID: 1,
+						},
+						{
+							ReceiverShardID: 2,
+						},
+					},
 				},
 			},
-		}, nil, outport.HeaderGasConsumption{}, 0, &outport.Pool{})
+			SignersIndexes:       []uint64{0, 1, 2},
+			TransactionPool:      &outport.TransactionPool{},
+			HeaderGasConsumption: &outport.HeaderGasConsumption{},
+		},
+	}
+	dbBlock, err := bp.PrepareBlockForDB(outportBlockWithHeader)
 	require.Nil(t, err)
 
 	expectedBlock := &data.Block{
 		Hash:                  "68617368",
 		Validators:            []uint64{0x0, 0x1, 0x2},
 		EpochStartBlock:       false,
 		SearchOrder:           0x3fc,
-		MiniBlocksHashes:      []string{"c57392e53257b4861f5e406349a8deb89c6dbc2127564ee891a41a188edbf01a", "28fda294dc987e5099d75e53cd6f87a9a42b96d55242a634385b5d41175c0c21"},
+		MiniBlocksHashes:      []string{"0796d34e8d443fd31bf4d9ec4051421b4d5d0e8c1db9ff942d6f4dc3a9ca2803", "4cc379ab1f0aef6602e85a0a7ffabb5bc9a2ba646dc0fd720028e06527bf873f"},
 		NotarizedBlocksHashes: []string(nil),
-		Size:                  104,
+		Size:                  230,
```

Comment:
size is calculated considering header and body which has not changed, right? or is it a different marshaling now?